### PR TITLE
.mailmap: Add entries for inconsistent users

### PR DESCRIPTION
**Pull request by q384566678**
Similar to https://github.com/opencontainers/runtime-tools/pull/115

Before this commit:
```
$ git shortlog -se 
  
     5  Aleksa Sarai <asarai@suse.de>
     1  Aleksa Sarai <cyphar@cyphar.com>
  ...
    38  Brandon Philips <brandon.philips@coreos.com>
     1  Brandon Philips <brandon@ifup.org>
  ...
     3  Doug Davis <dug@us.ibm.com>
     1  Doug Davis <duglin@users.noreply.github.com>
  ...
     3  Sergiusz Urbaniak <sergiusz.urbaniak@gmail.com>
    15  Sergiusz Urbaniak <sur@coreos.com>
  ...
    27  Stephen Day <stevvooe@users.noreply.github.com>
    15  Stephen J Day <stephen.day@docker.com>
 ...
     3  xiekeyang <keyang.xie@gmail.com>
    18  xiekeyang <xiekeyang@huawei.com>
```
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Brandon Philips <brandon.philips@coreos.com> <brandon@ifup.org>
+Doug Davis <dug@us.ibm.com> <duglin@users.noreply.github.com>
+Sergiusz Urbaniak <sergiusz.urbaniak@gmail.com> <sur@coreos.com>
+xiekeyang <keyang.xie@gmail.com> <xiekeyang@huawei.com>
```
Comment (wking) [inline .mailmap]: From your [PR comment][1], @xiekeyang uses the Huawei address more frequently than the Google address.  He might prefer you swap the order of addresses here to make the Huawei address canonical.

Similarly with @s-urbaniak in the line above.  You have the CoreOS address as more common, but the line above is making the Google address canonical.  And again for the @philips line.

Anyhow, the email these folks consider canonical is up to them, and we can always adjust the canonical choice if we hear from them later, but I'd have guessed a few of them the other way around ;).
https://github.com/opencontainers/image-tools/pull/186#issue-260471725

Comment (q384566678) [reply .mailmap]: updated.

File: .mailmap [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Aleksa Sarai <asarai@suse.de> <cyphar@cyphar.com>
+Brandon Philips <brandon.philips@coreos.com> <brandon@ifup.org>
+Doug Davis <dug@us.ibm.com> <duglin@users.noreply.github.com>
+Sergiusz Urbaniak <sur@coreos.com> <sergiusz.urbaniak@gmail.com>
+Stephen Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>
```
Comment (wking) [inline .mailmap]: From your [initial comment][1], @stevvooe uses “Stephen J Day” for his Docker address.  That's why I [suggested][2]:

    Stephen J Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>

Without the “J”, you get:

```
$ git describe
v0.2.0-43-g380a18c
$ git shortlog -se | grep Stephen
    27  Stephen Day <stephen.day@docker.com>
    15  Stephen J Day <stephen.day@docker.com>
```

With the line I suggest you get:

```
$ git shortlog -se | grep Stephen
    42  Stephen J Day <stephen.day@docker.com>
```

[1]: https://github.com/opencontainers/image-tools/pull/186#issue-260471725
[2]: https://github.com/opencontainers/image-tools/pull/186#pullrequestreview-65098077

File: .mailmap [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+xiekeyang <xiekeyang@huawei.com> <keyang.xie@gmail.com>
```
Comment (wking) [inline .mailmap]: I think you [still][1] want an additional line for `xiekeyang@users.noreply.github.com`.  As it stands you get:

```
$ git describe 
v0.2.0-43-g380a18c
$ git shortlog -se | grep xiekeyang
    21  xiekeyang <xiekeyang@huawei.com>
    22  xiekeyang <xiekeyang@users.noreply.github.com>
```

With an additional line you can collapse those together:

```
$ ENTRIES=$(echo 'xiekeyang <xiekeyang@huawei.com> <xiekeyang@users.noreply.github.com>'; cat .mailmap)
$ echo "${ENTRIES}" | sort --ignore-case | uniq >.mailmap
$ git shortlog -se | grep xiekeyang
    43  xiekeyang <xiekeyang@huawei.com>
```

[1]: https://github.com/opencontainers/image-tools/pull/186#pullrequestreview-65098077

Comment (q384566678) [reply .mailmap]: Done.

File: .mailmap [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Aleksa Sarai <asarai@suse.de> <cyphar@cyphar.com>
+Brandon Philips <brandon.philips@coreos.com> <brandon@ifup.org>
+Doug Davis <dug@us.ibm.com> <duglin@users.noreply.github.com>
+Sergiusz Urbaniak <sur@coreos.com> <sergiusz.urbaniak@gmail.com>
+Stephen J Day <stephen.day@docker.com> <stevvooe@users.noreply.github.com>
+xiekeyang <xiekeyang@huawei.com> <keyang.xie@gmail.com> <xiekeyang@users.noreply.github.com>
```
Comment (wking) [inline .mailmap]: I don't think this two-commit-emails syntax is supported (I don't see it discussed in `git-shortlog(1)` anyway).  So I'm getting:

```
$ git --version
git version 2.1.0.60.g85f0837
$ git describe
v0.2.0-43-gb76b86e
$ git shortlog -se | grep keyang
    21  xiekeyang <xiekeyang@huawei.com>
    22  xiekeyang <xiekeyang@users.noreply.github.com>
```

I think you want two entries:

```
xiekeyang <xiekeyang@huawei.com> <keyang.xie@gmail.com>
xiekeyang <xiekeyang@huawei.com> <xiekeyang@users.noreply.github.com>
```

**Comment by q384566678** [comment]
@coolljt0725 @xiekeyang PTAL

**Comment by coolljt0725** [comment]
LGTM

**Pull request merged**